pkg/reconciler/autoscaling/kpa/scaler.go

```diff
@@ -96,6 +97,8 @@ type scaler struct {
 func newScaler(ctx context.Context, psInformerFactory duck.InformerFactory, enqueueCB func(interface{}, time.Duration)) *scaler {
 	logger := logging.FromContext(ctx)
 	transport := pkgnet.NewProberTransport()
+	listerCache := make(map[schema.GroupVersionResource]cache.GenericLister, 1)
```

Comment:
Why 1? Could there be more than 1 possible values for GVR in this case?

Reply:
Right now no, only deployments. But we know there's gonna be at least one.